src/koala/__init__.py

```diff
@@ -572,11 +572,13 @@ def correct_high_cosmics_and_defects(
 
         if plot_suspicious_fibres == True and len(suspicious_fibres) > 0:
             # Plotting suspicious fibres..
-            figures = plot_suspicious_fibres(suspicious_fibres,
-                                   fig_size,
-                                   wave_min,
-                                   wave_max,
-                                   intensity_corrected_fiber=self.intensity_corrected)
+            figures = plot_suspicious_fibres_graph(
```

Comment:
This change references #15 where plot_suspicious_fibres was introduced as a plotting function, overwriting the bool option